src/Components/test/testassets/ComponentsApp.App/Pages/_Imports.razor

```diff
@@ -1 +0,0 @@
-@layout MainLayout
```

Comment:
I think I already know the answer, but this somewhat de-emphasizes `@layout`. What example would we show/put in the template? 

I think we'd probably use the router-based approach and reserve `@layout` for overrides.

Reply:
> I think we'd probably use the router-based approach and reserve @layout for overrides.

Exactly. I think `DefaultLayout` more naturally matches what people are trying to do, and `@layout` has some unexpected caveats like blowing up if the layout component itself is in the same folder as `_Imports.razor`.

I'd consider `@layout` to be a feature for overriding the default layout only.

Reply:
Great, I think this is an overall improvement.